Main_Data_Class_2.rb

```diff
@@ -0,0 +1,7 @@
+# 2. There is a string. Find a maximum number of digits placed in a row in it
```

Comment:
'Find maximum numbers of digits placed in a row' means that we shouldn't find a maximum number in a string but we should find a maximum number of chars placed in a row where each is a digit. For example:
the string is 'lkj5d54d454d5454654d654d6546'
your method should return '6' as an answer since we have here few groups of digits and fourth is the biggest one